<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Responsible</class>
 <widget class="QWidget" name="Responsible">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ответственные</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelItemFirstName">
       <property name="text">
        <string>Имя</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditItemFirstName">
       <property name="inputMask">
        <string>&gt;A&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelItemSurname">
       <property name="text">
        <string>Фамилия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditItemSurname">
       <property name="inputMask">
        <string>&gt;A&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelItemLastName">
       <property name="text">
        <string>Отчество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditItemLastName">
       <property name="inputMask">
        <string>&gt;A&lt;aaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPosition">
       <property name="text">
        <string>Должность</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxPositions">
       <item>
        <property name="text">
         <string>Руководитель отдела</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Старший кладовщик</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Кладовщик</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Товаровед</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonItemInsert">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutEditing">
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdateCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
